### PR TITLE
Feature: Moderator can revoke moderator role to others and himself

### PR DESCRIPTION
**Pull request by BenjaminVega**

### CODE REVIEW

File: JitsiConference.js [hunk 1]
```diff
@@ -1470,6 +1470,23 @@ JitsiConference.prototype.grantOwner = function(id) {
...
+ * @param {string} id id of the participant to revoke owner rights to.
+ */
+JitsiConference.prototype.revokeOwner = function(id) {
+    const participant = this.getParticipantById(id);
+    const isMyself = this.myUserId() === id;
+
+    if (isMyself) {
+        this.room.setAffiliation(this.room.myroomjid, 'member');
+    } else if (participant) {
+        this.room.setAffiliation(participant.getJid(), 'member');
```
Comment (damencho) [inline JitsiConference.js:1485]: We use `member` only when lobby is enabled in the rest of the cases it is `none`.

Comment (BenjaminVega) [reply JitsiConference.js]: Oh, I see. I have pushed a commit to deal with that.
Thanks! 😄

File: JitsiConference.js [hunk 2]
```diff
@@ -1470,6 +1470,24 @@ JitsiConference.prototype.grantOwner = function(id) {
...
 
+/**
+ * Revoke owner rights to the participant or local Participant as
+ * the user might want to refuse to be a moderator.
+ * @param {string} id id of the participant to revoke owner rights to.
+ */
+JitsiConference.prototype.revokeOwner = function(id) {
+    const participant = this.getParticipantById(id);
+    const isMyself = this.myUserId() === id;
+    const role = this.enableLobby() ? 'member' : 'none';
```
Comment (damencho) [inline JitsiConference.js:1481]: This actually enables lobby and returns promise ...

Comment (damencho) [reply JitsiConference.js:1481]: You should use this.isMembersOnly()

Comment (BenjaminVega) [reply JitsiConference.js]: fixed

**Review by damencho — APPROVED** [review]

**Comment by damencho** [comment]
As a next step, you should add docs for the new method :) here https://github.com/jitsi/handbook/blob/master/docs/dev-guide/ljm-api.md, thank you for your contribution.

**Pull request merged**